### PR TITLE
Improve OMZ demos for launching REDOWA models out-of-the-box

### PR DESCRIPTION
**Pull request by sivanov-work**
1) Add `scale/normalization` for some demos in command arguments
2) Enforce segmentation model layout to NCHW conversion

### CODE REVIEW

File: demos/common/cpp/models/src/image_model.cpp [hunk 1]
```diff
@@ -45,7 +45,9 @@ std::shared_ptr<InternalModelData> ImageModel::preprocess(const InputData& input
         const size_t height = tensorShape[ov::layout::height_idx(layout)];
         const size_t channels = tensorShape[ov::layout::channels_idx(layout)];
         if (static_cast<size_t>(img.channels()) != channels) {
-            throw std::runtime_error("The number of channels for model input and image must match");
+            throw std::runtime_error(std::string("The number of channels for model input: ") +
+                                     std::to_string(channels) + " and image: " +
+                                     std::to_string(static_cast<size_t>(img.channels())) + " - must match");
```
Comment (Wovchena) [inline demos/common/cpp/models/src/image_model.cpp:50]:
```suggestion
                                     std::to_string(img.channels()) + " - must match");
```
Can't `to_string()` construct from int?

File: demos/common/cpp/models/src/segmentation_model.cpp [hunk 2]
```diff
@@ -69,7 +69,7 @@ void SegmentationModel::prepareInputsOutputs(std::shared_ptr<ov::Model>& model)
     }
 
     ov::preprocess::PrePostProcessor ppp(model);
-    ppp.input().tensor().set_element_type(ov::element::u8).set_layout({"NHWC"});
+    inputTransform.setPrecision(ppp, model->input().get_any_name());
```
Comment (Wovchena) [inline demos/common/cpp/models/src/segmentation_model.cpp:72]:
```suggestion
    inputTransform.setPrecision(ppp, model->input().get_any_name());
    ppp.input().tensor().set_layout("NHWC");
```
You've lost `set_layout({"NHWC"})`. Put it back

**Comment by Wovchena** [comment]
`Cpp Demos (Ubuntu 20.04)` and ` Python Demos (Ubuntu 20.04)` failures are known. Ignore them

File: demos/common/cpp/models/src/image_model.cpp (hunk 1 above)
Comment (sivanov-work) [reply demos/common/cpp/models/src/image_model.cpp]: fair enough, applied

**Review by Wovchena — APPROVED** [review]

**Pull request merged**